gensim/models/fasttext.py

```diff
@@ -45,6 +81,7 @@
 logger = logging.getLogger(__name__)
 
 try:
+    raise ImportError
```

Comment:
???

Reply:
I wanted to make sure that doctests work even without numpy installed so I did this hack to force my computer to use the python implementation. And then i forgot to remove it!